src/index.ts

```diff
@@ -50,7 +50,7 @@ async function handleRequest(request: Request) {
     }
 
   const body = JSON.stringify({
-    content: `New changes were made in ${projectName}!\n${updates.join('\n')}`,
+    content: `New changes were made in ${projectName}!\n${updates.join('\n')} \nLink: https://crwd.in/${res.events[0].project}`,
```

Comment:
Maybe we can do something like this?

```suggestion
content: [
  `New changes were made in ${projectName!}`,
  ...updates,
  `Link to project: ${res.events[0].project}`,
].json("\n"),
```

Reply:
content is not used anymore, message being sent as embed